packages/api-explorer/src/lib/oas-to-har.js

```diff
@@ -121,6 +121,20 @@ module.exports = (
     });
   }
 
+  // Add custom query params
+  const {query} = formData
+  if (query && Object.keys(query).length > har.queryString.length) {
```

Comment:
Why this check? Does formData always contain the queryparams defined in the har?

Isn't this covered by the filter you apply at line 129.

I worry that it may happen that some query param si not filled and thus does not appear to be in formData while there are others and the length might be `<=`..

I'd run through all the formData and verify whether each key exists or not

Reply:
This check is mainly to avoid further check when we are calling POST apis. In the POST call the query object is an empty object so we can avoid some computations. (From API portal we cannot add query params)

The formData is equal to the input `values` of the funcion. I will verify it.

Reply:
formData può essere modificato solamente da Doc.js riga 134: 
`        formData: Object.assign({}, previousState.formData, {[typeFromSchema]: filtered}),`

quindi `{ query: {}}` mi aspetto che sia sempre filtrato bene